path/src/cmd/logo.ps1

```
@@ -0,0 +1,19 @@
+﻿function script:cmd_logo {
+	require deno
+	$iconAnime = "$FOUNT_DIR/imgs/icon_anime/index.mjs"
+	$originalTitle = Get-Title
+	try {
+		Set-Title '𝒻ℴ𝓊𝓃𝓉 𝓵𝓸𝓰𝓸'
+		$args = @($args | Select-Object -Skip 1)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)